packages/styling/src/styles/theme.ts

```diff
@@ -38,27 +40,29 @@ export function getTheme(): ITheme {
 }
 
 /**
- * Loads the default global theme definition.
+ * Applies the theme, while filling in missing slots.
  */
-export function loadTheme(theme: Partial<ITheme>): void {
+export function loadTheme(theme: IPartialTheme): ITheme {
   _theme = createTheme(theme);
 
   // Load the legacy theme from the palette.
   legacyLoadTheme(_theme.palette as {});
 
   GlobalSettings.setValue(ThemeSettingName, _theme);
+
+  return _theme;
 }
 
 /**
  * Creates a custom theme definition which can be used with the Customizer.
  */
-export function createTheme(theme: Partial<ITheme>): ITheme {
-  let newPalette = { ..._theme.palette, ...theme.palette };
+export function createTheme(theme: IPartialTheme): ITheme {
+  let newPalette = { ...DefaultPalette, ...theme.palette };
 
   return {
     palette: newPalette,
     fonts: {
-      ..._theme.fonts,
+      ...DefaultFontStyles,
```

Comment:
I don't know why you changed this. Technically, _theme.fonts should be whatever was previously set. Why reset back here? This change doesn't make sense to me.